### DISCUSSION TITLE
Feature: Allow 3D subclaims

### DISCUSSION THREAD

**Discussion by Kycermann** [GriefPrevention/GriefPrevention#1376] [Ideas]
**As** someone who has a vertically extending base
**I want** to give public containertrust in one room but not the room above it
**Because** one room has community / free items in it and the other has my private valuables

I'm frustrated when I want to give access to one chest and have to watch out not to place any chests above/below it.

We could have a new claims mode that specifically does 3D claims, like /ydivideclaims

**Comment by Jikoo** [comment]
See #269
